pkgs/top-level/python-packages.nix

```
@@ -4513,8 +4513,14 @@ self: super: with self; {
 
   grpcio = callPackage ../development/python-modules/grpcio { };
 
+  grpcio-channelz = callPackage ../development/python-modules/grpcio-channelz { };
+
+  grpcio-health-checking = callPackage ../development/python-modules/grpcio-health-checking { };
+
   grpcio-gcp = callPackage ../development/python-modules/grpcio-gcp { };
```

```suggestion
  grpcio-gcp = callPackage ../development/python-modules/grpcio-gcp { };

  grpcio-health-checking = callPackage ../development/python-modules/grpcio-health-checking { };
```
nit
Besides this, it looks good to me.